sputnikdao2/src/proposals.rs

```diff
@@ -250,20 +251,23 @@ impl Contract {
                     ONE_YOCTO_NEAR,
                     GAS_FOR_FT_TRANSFER,
                 )
-                .into()
             }
+            .into()
         }
     }
 
+    fn internal_return_bond(&self, policy: &Policy, proposal: &Proposal) -> Promise {
+        Promise::new(proposal.proposer.clone()).transfer(policy.proposal_bond.0)
```

Comment:
At some point I would like to subtract the storage cost associated with the proposal, but that's not for this PR and captured in #47